easybuild/easyconfigs/c/cURL/cURL-7.40.0-GCC-4.9.2.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.40.0'
+
+homepage = 'http://curl.haxx.se'
+description = """libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS,
+ POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP. libcurl supports
+ SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP form based upload,
+ proxies, cookies, user+password authentication (Basic, Digest, NTLM, Negotiate,
+ Kerberos), file transfer resume, http proxy tunneling and more."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://curl.haxx.se/download/']
+
+dependencies = [('OpenSSL', '1.0.1k')]
```

Comment:
an `openssl` provided by the OS should be preferred here, cfr. the Python easyconfigs?